### PR TITLE
Adds Jest to Rogue

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- Adds Jest and Babel Jest to Rogue
- Sets up a test to make sure it is properly set up. 

#### How should this be reviewed?
👀  and run `npm test`. Tests should work and pass: 
screen shot 2017-09-12 at 11 52 58 am

#### Relevant tickets
Fixes https://www.pivotaltracker.com/n/projects/2019429/stories/150971771

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "start": "npm run clean && NODE_ENV=development webpack --watch",
     "build": "npm run clean && NODE_ENV=production webpack",
     "build:dev": "npm run clean && NODE_ENV=development webpack",
-    "clean": "rm -rf public/dist"
+    "clean": "rm -rf public/dist",
+    "test": "jest"
   },
   "babel": {
     "presets": [
```
Comment (chloealee) [inline package.json]: @DFurnes should I also add `flow` in the presets similar to [Phoenix Next](https://github.com/DoSomething/phoenix-next/blob/dev/package.json#L22)? 

Also, I think I can skip adding 
```
{
  "presets": ["es2015", "react"]
}
``` 
like it says [here (under Additional Configuration](https://facebook.github.io/jest/docs/en/getting-started.html#content)?

File: package.json [hunk 2]
```diff
@@ -23,6 +24,9 @@
   "devDependencies": {
     "@dosomething/babel-config": "^1.0.0",
     "@dosomething/webpack-config": "^2.0.4",
+    "babel-jest": "^21.0.2",
+    "jest": "^21.0.2",
+    "regenerator-runtime": "^0.11.0",
```
Comment (chloealee) [inline package.json]: @DFurnes [This](https://facebook.github.io/jest/docs/en/getting-started.html#content) said to also install `regenerator-runtime` but I don't think PN has it - is this needed?

File: package.json (hunk 1 above)
Comment (DFurnes) [reply package.json]: We're not using Flow in Rogue at the moment, so nope! Safe to leave out. And yep, can leave the Babel configuration as is. 😄

File: package.json (hunk 2 above)
Comment (DFurnes) [reply package.json]: Nope, we should be using npm 4+ everywhere so no need.

File: package.json (hunk 1 above)
Comment (chloealee) [reply package.json]: awesome, thanks @DFurnes !

File: package.json (hunk 2 above)
Comment (chloealee) [reply package.json]: Ok cool - deleted!

**Comment by DFurnes** [comment]
Let's also [add this to our Wercker builds](https://github.com/DoSomething/phoenix-next/blob/ff451405aabea585a34916daa6636fb119f2c2dc/wercker.yml#L13-L15), so we can make sure tests pass before merging PRs.

In `wercker.yml`:

```yml
- script:
    name: test javascript code
    code: npm run test:ci
```

And in `package.json`, a [CI-specific Jest task](https://github.com/DoSomething/phoenix-next/blob/ff451405aabea585a34916daa6636fb119f2c2dc/package.json#L5) (that runs tests non-interactively):

```json
"test:ci": "NODE_ENV=test && jest",
```

**Comment by chloealee** [comment]
@DFurnes ah so cool!

**Comment by DFurnes** [comment]
This looks great @chloealee! 🚥 🎊

**Pull request merged**